migrate_repo/versions/008_Add_shipped_at.py

```diff
@@ -0,0 +1,25 @@
+# Upgrade/downgrade the database with the shippedAt field.
+# CF bug #1101596 for more information
+from sqlalchemy import Column, MetaData, Table, DateTime
+
+import pytz
+
+from datetime import datetime
+
+
+def upgrade(migrate_engine):
+    metadata = MetaData(bind=migrate_engine)
+
+    def add_shippedAt(table):
+        shippedAt = Column('shippedAt', DateTime(pytz.utc), default=datetime.utcnow)
+        shippedAt.create(table)
+    add_shippedAt(Table('fennec_release', metadata, autoload=True))
+    add_shippedAt(Table('firefox_release', metadata, autoload=True))
+    add_shippedAt(Table('thunderbird_release', metadata, autoload=True))
```

Comment:
What's your plan/hope for filling in historical data accurately? This upgrade method is going to end up with every existing release having an inaccurate shipped date, obviously.